extensions/podman/packages/extension/src/extension.ts

```diff
@@ -2317,3 +2317,7 @@ export function updateWSLHyperVEnabledContextValue(value: boolean): void {
     extensionApi.context.setValue(WSL_HYPERV_ENABLED_KEY, value);
   }
 }
+
+export function setStopLoop(value: boolean): void {
+  stopLoop = value;
```

Comment:
maybe the stopLoop should be reset in deactivate method

I am not a big fan of exposing a method for test purpose only :/

Reply:
Oops sorry I thought adding stop loop in deactivate.

You meant, calling deactivate from the test, I can do it!

Reply:
When we deactivate an extension we want to dispose its resources, and stop its activity, when calling deactivate I expect the stopLoop to be true, and when reactivating I expect it to be set to false

Reply:
Adding stopLoop = false in activate to make this change possible
https://github.com/podman-desktop/podman-desktop/pull/13275